src/rpc/evmRPCSender.ts

```diff
@@ -0,0 +1,71 @@
+import {CHAINID} from 'apy-vision-config';
+import {RPCOracle} from "@src/rpc/rpcOracle";
+import {ArchiveLogger, REQUEST_ID} from "@src/logger";
+import {performance} from "perf_hooks";
+import {asL2Provider} from "@eth-optimism/sdk";
+import {ethers} from "ethers";
+import {KafkaManager} from "logging-library";
+import {Logger} from 'log4js';
+import {AbstractRPCSender} from "@src/rpc/abstractRPCSender";
+
+
+export class EvmRPCSender extends AbstractRPCSender {
```

Comment:
previously it was bad encapsulation as everything was in the utility file -- the work is to extract this class to better encapsulate logic